src/Uno.UI/UI/Xaml/Data/Binding.cs

```diff
@@ -129,7 +127,7 @@ public object FallbackValue
 		/// Gets or sets a value that indicates the direction of the data flow in the binding.
 		/// </summary>
 		/// <value>The mode.</value>
-		public BindingMode Mode { get; set; }
+		public BindingMode Mode { get; set; } = BindingMode.OneWay;
```

Comment:
Is this a breaking change?

Reply:
Nope. Previously, it defaulted to `(BindingMode)0` which isn't valid for BindingMode. But all our reads of BindingMode compare explicitly against TwoWay, so such comparisons will still be the same result.